site/source/tutorials/001_static_content/demo/finished/biz-e-corp/tests/unit/widgets/HelloWorld.ts

```diff
@@ -9,7 +9,8 @@ registerSuite({
 		const helloWorld = new HelloWorld();
 
 		const vnode = <VNode> helloWorld.__render__();
-		assert.strictEqual(vnode.vnodeSelector, 'div');
-		assert.equal(vnode.text, 'Hello, Dojo World!');
+		assert.strictEqual(vnode.vnodeSelector, 'h1');
+		assert.equal(vnode.text, 'Biz-E Bodies');
+		assert.equal(vnode.properties.title, 'I am a title!');
```

Comment:
@agubler no doubt -- I just think it reads more clearly if the assertion is `vnode.properties && vnode.properties.title`. To someone who's read a lot more JS than TS, anyway... Can't help my inclination towards the more JS-like option. I really appreciate the explanation though. I'm sure it'll come in handy in the future. 👍